src/ol/interaction/interaction.js

```diff
@@ -181,6 +182,14 @@ ol.interaction.Interaction.zoomByDelta = function(view, delta, opt_anchor, opt_d
   var currentResolution = view.getResolution();
   var resolution = view.constrainResolution(currentResolution, delta, 0);
 
+  if (resolution !== undefined) {
+    var resolutions = view.getResolutions();
+    resolution = ol.math.clamp(
+        resolution,
+        view.getMinResolution() || resolutions[resolutions.length - 1],
+        view.getMaxResolution() || resolutions[0]);
+  }
```

Comment:
Previously, we would let people temporarily zoom outside of the min/max range.  This was intentional.  When using a trackpad for scrolling, for example, it feels nice to have the map temporarily zoom in/out beyond the constraints and then settle back to the constrained value.  This change breaks that behavior.

Reply:
That might not be true...

Reply:
Looks like this isn't the case.  Ignore the noise.

Reply:
Thanks for the mention. I've got to check on mac device but without that condition, it doesn't work as expected for setMin/MaxZoom.